deploy/warehouse/target/data.sql

```diff
@@ -5,7 +5,17 @@ begin;
 
 insert into warehouse.organism (lineage, identifiers, details)
     values
+        ('Adenovirus.1',            null,                       null),
+        ('Adenovirus.2',            null,                       null),
         ('Bordetella_pertussis',    'NCBITAXON => 520',         '{"report_to_public_health": true}'),
+        ('Chlamydophila_pneumoniae',null,                       null),
+        ('Coronavirus.229E_OC43',   null,                       null),
+        ('Coronavirus.HKU1_NL63',   null,                       null),
+        ('Enterovirus',             null,                       null),
+        ('Enterovirus.D68',         null,                       null),
```

Comment:
Should be `Enterovirus.D.68` since enterovirus D is the species and 68 is the subtype